templates/admin/_sidenav.html.twig

```diff
@@ -30,8 +30,6 @@
 		<li>
 			<a class="nav-link {{ _self.activeLink('admin_streamers') }}" href="{{ path('admin_streamers_index') }}"><i class="fab fa-fw fa-twitch mr-2 fa-2x"></i> Streamers</a>
 		</li>
-	{% endif %}
-	{% if is_granted('ROLE_SUPER_ADMIN') %}
```

Comment:
link should still be hidden for ROLE_REDACTEUR/ROLE_DEFAULT (using `is_granted(ROLE_ADMIN)` should be enough)

Reply:
This link is hidden for all roles except `ROLE_ADMIN` and `ROLE_SUPER_ADMIN` because it is in the `if` block between line 20 and 36.